packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTableRow/MyProfileCounterpartiesTableRow.tsx

```diff
@@ -7,11 +7,15 @@ import './MyProfileCounterpartiesTableRow.scss';
 
 type TMyProfileCounterpartiesTableRowProps = {
     id: string;
-    isBlocked: boolean;
+    is_blocked: boolean;
     nickname: string;
 };
 
-const MyProfileCounterpartiesTableRow = ({ id, isBlocked, nickname }: TMyProfileCounterpartiesTableRowProps) => {
+const MyProfileCounterpartiesTableRow = ({
+    id,
+    is_blocked: isBlocked,
```

Comment:
```suggestion
    isBlocked,
```

Reply:
I think i left these cause there was some issues with the return values of the data, cause its returning is_blocked and not isBlocked, but I'll see if theres a workaround